cbinfoblox/bridge.py

```diff
@@ -23,10 +24,10 @@
 
 import version
 
-logging.getLogger("requests").setLevel(logging.WARNING)
+logging.getLogger("requests").setLevel(logging.DEBUG)
```

Comment:
revert this change so we don't spam logs with debugging information from requests